src/components/utility/ViewDrawerOrModal.tsx

```diff
@@ -17,31 +17,32 @@ const ViewDrawerOrModal = ({
    isMobile,
    children,
 }: ViewDrawerOrModalProps) => {
+   if (!isOpen) return null;
    return isMobile ? (
       <div>
          <Drawer
             open={isOpen}
             onClose={onClose}
             position='bottom'
             theme={bottomSheetDrawerTheme}
-            className='h-[calc(100%-75px)]'
+            className='h-[calc(100%-75px)] rounded-t-xl'
          >
             <Drawer.Header
                title={title}
                titleIcon={() => null}
                closeIcon={() => <XMarkIcon className='h-8 w-8' />}
             />
-            <Drawer.Items key={isOpen.toString()} className='flex flex-col h-full'>
+            <Drawer.Items key={isOpen.toString()} className='flex flex-col h-full mb-12'>
                {children}
             </Drawer.Items>
          </Drawer>
       </div>
    ) : (
-      <Modal show={isOpen} onClose={onClose} size={'sm'}>
+      <Modal show={isOpen} onClose={onClose} size={'sm'} style={{ outline: 'none' }}>
```

Comment:
inline style because the tailwind style `outline-none` does not remove the outline. The goal is to make it so that when you click the modal and then type, the outline does not appear
ref https://tailwindcss.com/docs/outline-style#removing-outlines

Reply:
I also tried my own class which didn't work either.

```css
.no-outline,
.no-outline:focus,
.no-outline:active {
   outline: none !important;
}
```

Reply:
The problem with outline-none is it doesn't actually set `outline: none`
As for checking the browser console, Idk how to check a focused state because as soon as I click anywhere else it unfocuses

Reply:
<img width="1728" alt="Screenshot 2024-11-12 at 21 34 27" src="https://github.com/user-attachments/assets/35a87dfd-0c79-4d2d-bc48-07659fd9cb5b">

Reply:
> The problem with outline-none is it doesn't actually set outline: none

weird. what does it do then?